src/codegen/riscv64/cpu-riscv64.cc

```diff
@@ -16,7 +16,8 @@ namespace internal {
 
 void CpuFeatures::FlushICache(void* start, size_t size) {
 #if !defined(USE_SIMULATOR)
-  __builtin___clear_cache(start, (char *)start + size);
+   //FIXME:builtin_clear_cache don't work, so replace it with 'fence.i'.
```

Comment:
To keep a consistent style, making it easier to find our fixmes, please change this to:
```
  // FIXME(RISCV): builtin_clear_cache doesn't work yet, so use `fence.i` for now
  // __builtin___clear_cache(start, (char *)start + size);
```

Reply:
Thank you for your advice. I have fixed it.